dashboard/app/controllers/projects_controller.rb

```diff
@@ -333,7 +333,8 @@ def show
       game_display_name: data_t("game.name", @game.name),
       app_name: Rails.env.production? ? t(:appname) : "#{t(:appname)} [#{Rails.env}]",
       azure_speech_service_voices: azure_speech_service_options[:voices],
-      disallowed_html_tags: disallowed_html_tags
+      disallowed_html_tags: disallowed_html_tags,
+      blocklyVersion: params[:blocklyVersion]
```

Comment:
I know this was kind of thrown in (which I am stoked to have!), are we confident it won't cause any issues?

Reply:
I feel confident. It works as expected across a variety of Blockly projects and doesn't cause any issues in non-Blockly projects such as App Lab or Game Lab. It's also just an optional flag that we'd never force any user to add to their URLs.